Mage/src/main/java/mage/game/draft/DraftImpl.java

```diff
@@ -209,20 +219,53 @@ protected boolean pickCards() {
                 return false;
             }
             player.setPicking();
-            player.getPlayer().pickCard(player.getBooster(), player.getDeck(), this);
+            player.setBoosterNotLoaded();
         }
-        cardNum++;
+        setupBoosterLoadingHandle();
         synchronized (this) {
             while (!donePicking()) {
                 try {
-                    this.wait();
+                    this.wait(3000);
                 } catch (InterruptedException ex) {
```

Comment:
Same comment, shouldn't this do something with the exception?

Reply:
I don't know, it was like that and I'm not going to touch it. I don't think the lack of exception handling is the cause of the issues here.

Reply:
Alright. I'll make a separate issue for it.